app/controllers/node_info_controller.rb

```diff
@@ -0,0 +1,24 @@
+class NodeInfoController < ApplicationController
+  respond_to :json
+  respond_to :html, only: :statistics
+
+  def jrd
+    render json: NodeInfo.jrd(CGI.unescape(node_info_url("123.123").sub("123.123", "%{version}")))
```

Comment:
Because of the constraint on the route. But true, I'm rather unhappy about  this "solution".